### PR TITLE
docs: ✏️ add list of recommended tooling

### PR DESCRIPTION
**Pull request by randallmorey**
This PR updates the readme with our team's common and recommended tooling.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,12 +46,19 @@ You will need the following things properly installed on your computer.
...
+
+## Optional Tooling
+
+Our team finds the following applications useful in day-to-day development
+workflows.  These are merely recommendations.  We encourage you to use the tools
+that best suit you and your personal style.
+
+* [VS Code][vscode]
+* [Warp][warp]:  Terminal application for macOS
+* [Homebrew][homebrew]:  The missing package manager for macOS
```
Comment (DhariniJeeva) [inline README.md:58]:
```suggestion
* [Homebrew][homebrew]:  The missing package manager for macOS and Linux
```

File: README.md [hunk 2]
```diff
@@ -45,12 +46,19 @@ You will need the following things properly installed on your computer.
...
-[firefox]: https://firefox.com/
-[yarn-workspaces]: https://classic.yarnpkg.com/en/docs/workspaces/
+
+## Optional Tooling
+
+Our team finds the following applications useful in day-to-day development
+workflows.  These are merely recommendations.  We encourage you to use the tools
+that best suit you and your personal style.
+
+* [VS Code][vscode]
```
Comment (DhariniJeeva) [inline README.md:56]:
```suggestion
* [VS Code][vscode]: Code editor for building and debugging web applications
```

**Review by DhariniJeeva — APPROVED** [review]
LGTM, Thank you for this!

**Review by dvonanderson — APPROVED** [review]
Awesome Stuff

**Review by calcaide — APPROVED** [review]
LGTM! Thank you

**Review by lisbet-alvarez — APPROVED** [review]
Nice! LGTM!

**Review by ZedLi — APPROVED** [review]

**Review by cameronperera — APPROVED** [review]

**Pull request merged**